src/Connection/MessageReceivers/BaseMessageReceiver.ts

```diff
@@ -24,7 +24,7 @@ export abstract class BaseMessageReceiver<TMessage> implements IBaseMessageRecei
     }
 
     /**
-     * The {@link ActionCode}s that are handled by this message receiver
+     * The {@link ActionCode | ActionCodes } that are handled by this message receiver
```

Comment:
👀 I think the trailing space on ActionCodes will be printed (resulting in two spaces) so should be
`The {@link ActionCode | ActionCodes} that`

Reply:
(Same for all)

Reply:
Updated